app/views/articles/show.html.haml

```diff
@@ -1,19 +1,24 @@
 .container.flex--2__2-1
   %article{ tabindex: '0', id: 'content' }
     %header
+      - if @collections.any?
+        %p= WrappersHelper.render(@collections)
+
       %h1= sanitize @article.title
       = MarkdownHelper.markdown(sanitize @article.article_summary).html_safe
 
     %div{ role: 'main' }
       = MarkdownHelper.markdown(sanitize @article.article_body).html_safe
-      = "Up to:"
-      - @collections.each do |collection|
-        = link_to(sanitize(collection.name), collection_path(collection.graph_id))
+
+    %footer
```

Comment:
This will be empty if none, do we want `%footer` in that instance? We may do, just checking...